### PR TITLE
crawler-youtube: video-status-api

### PR DESCRIPTION
**Pull request by dragonjet**

### CODE REVIEW

File: apps/crawler-youtube/tasks/video-status-api.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    log.debug('videoStatusAPI() START');
+
+    // Get times
+    const utcDate = moment.tz('UTC');
+
+    // Check if there are live videos
+    const liveVideos = await db.Video.findAll({
+      where: [
+        { yt_video_key: { [Op.not]: null } },
+        { status: consts.STATUSES.LIVE },
```
Comment (IAmVisco) [inline apps/crawler-youtube/tasks/video-status-api.js:27]: Use newer const destruct.

File: apps/crawler-youtube/tasks/video-status-api.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            }
+          } else {
+            saveInfo.status = consts.STATUSES.PAST;
+          }
+          // Get derived values
+          if (saveInfo.live_schedule && saveInfo.live_start) {
+            saveInfo.late_secs = parseInt(startMoment.diff(scheduleMoment, 'seconds'), 10);
+          }
+          if (saveInfo.live_end && saveInfo.live_start) {
+            saveInfo.duration_secs = parseInt(endMoment.diff(startMoment, 'seconds'), 10);
```
Comment (IAmVisco) [inline apps/crawler-youtube/tasks/video-status-api.js:141]: Parse int is extra, `.as()` returns number.

File: apps/crawler-youtube/tasks/video-status-api.js (hunk 1 above)
Comment (dragonjet) [reply apps/crawler-youtube/tasks/video-status-api.js]: fixed as of latest

File: apps/crawler-youtube/tasks/video-status-api.js (hunk 2 above)
Comment (dragonjet) [reply apps/crawler-youtube/tasks/video-status-api.js]: fixed as of latest

File: apps/crawler-youtube/tasks/video-status-api.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+module.exports = async () => {
+  try {
+    log.debug('videoStatusAPI() START');
+
+    // Get times
+    const utcDate = moment.tz('UTC');
+
+    // Check if there are live videos
+    const targetVideos = await db.Video.findAll({
+      where: [
```
Comment (sphinxrave) [inline apps/crawler-youtube/tasks/video-status-api.js:25]: do a object here please, not sure what an Array does.

**Pull request merged**